client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -0,0 +1,204 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <div class="container">
+        <div class="recovery-container">
+          <div v-show="state === ExportDevicePageState.Start">
+            <ms-informative-text>
+              {{ $t('RecoveryDevicePage.subtitles.newPassword') }}
+            </ms-informative-text>
+            <ms-informative-text>
+              {{ $t('RecoveryDevicePage.subtitles.twoFilesToKeep') }}
+            </ms-informative-text>
+
+            <div>
+              <div class="block">
+                {{ $t('RecoveryDevicePage.titles.recoveryFile') }}
+              </div>
+              <div class="block">
+                {{ $t('RecoveryDevicePage.titles.secretKey') }}
```

Comment:
Should be `ExportRecoveryDevicePage` or something like that, since we'll have another page for the import, it may be confusing.